<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RedPill</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../redpill.png</normaloff>../redpill.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="fileTextEdit">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectFileButton">
       <property name="text">
        <string>选择文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="text">
        <string>清除</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>801</width>
      <height>541</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="minimumSize">
        <size>
         <width>510</width>
         <height>531</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>510</width>
         <height>531</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>API名/参数</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>详细信息</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="processNameLable">
           <property name="text">
            <string>进程名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="processName">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="processIdLabel">
           <property name="text">
            <string>进程ID：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="processId">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="processPriorityLabel">
           <property name="text">
            <string>进程权限：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="processPriority">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="processInfoLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>进程信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="processInfoTextEdit">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="warningButton">
         <property name="text">
          <string>警报</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="warningTextEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>804</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
